plugins/composer/composer.plugin.zsh

```diff
@@ -51,5 +51,10 @@ alias cgrm='composer global remove'
 # install composer in the current directory
 alias cget='curl -s https://getcomposer.org/installer | php'
 
-# Add Composer's global binaries to PATH
-export PATH=$PATH:$(composer global config bin-dir --absolute) 2>/dev/null
+# Add Composer's global binaries to PATH, using Composer if available.
+if hash composer 2>/dev/null; then
+  export PATH=$PATH:$(composer global config bin-dir --absolute) 2>/dev/null
```

Comment:
The `2>/dev/null` part should be *inside* the parenthesis to remove an unwanted info message, like this:

`export PATH=$PATH:$(composer global config bin-dir --absolute 2>/dev/null)`

Reply:
This line is the same as it was. Should we be correcting that line in this PR?

Reply:
Fixed it as part of this.